src/url-parse-query-string.js

```diff
@@ -37,7 +37,9 @@ export function parseQueryString_(queryString) {
   let match;
   while ((match = regex.exec(queryString))) {
     const name = tryDecodeUriComponent_(match[1], match[1]);
-    const value = match[2] ? tryDecodeUriComponent_(match[2], match[2]) : '';
+    const value = match[2]
+      ? tryDecodeUriComponent_(match[2].replace(/\+/g, '%20'), match[2])
```

Comment:
```suggestion
      ? tryDecodeUriComponent_(match[2].replace(/\+/g, ' '), match[2])
```

Reply:
I considered this as well, and ultimately it doesn't matter, but my rationale for using `%20` is that match[2] is supposedly in an encoded state before running through `tryDecodeUriComponent_`, so inserting `%20` at this stage just means the string being decoded will mostly like be uniformly encoded.

Reply:
It's a bit more work for the decoder to decode, and this'll save 2 bytes. It's nit-picky.

Reply:
Resolved with commit 6ac80ee